satpy/resample.py

```diff
@@ -148,13 +148,11 @@
 
 import dask.array as da
 import numpy as np
-import pyresample
 import xarray as xr
 import zarr
-from packaging import version
 from pyresample.ewa import DaskEWAResampler, LegacyDaskEWAResampler
 from pyresample.geometry import SwathDefinition
-from pyresample.gradient import GradientSearchResampler
+from pyresample.gradient import create_gradient_search_resampler
```

Comment:
The module docstring still uses the class. If it shouldn't, could you update it to point to this function too?